CMakeLists.txt

```diff
@@ -1,88 +1,45 @@
-cmake_minimum_required(VERSION 3.16)
-project(jluna)
+cmake_minimum_required(VERSION 3.12)
 
-set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_BUILD_TYPE Debug)
+project(jluna VERSION 0.0.0 LANGUAGES CXX)
```

Comment:
```suggestion
project(jluna VERSION 8.4.0 LANGUAGES CXX)
```

Reply:
jluna has been through 8 major version releases? I just noticed actually that the readme says v0.8.1.